File: Python/Interpreter.py
'''File name: Interpreter.py
Created by Joshua Brock on 10/25/2018'''

from Memory import *
from Exception import Exception
from Parser import Parser

if __name__ == '__main__':
    '''Our main method to run our code through test files'''
    try:
        p = Parser('[[[insert file location here]]]')
        program = p.parse()
        program.execute()
        displayMemory()
        
    except Exception as e:
        print(e.message)

File: Java/Interpreter.java
//File name: Interpreter.java
//Created by Joshua Brock on 9/24/2018

import java.io.FileNotFoundException;


public class Interpreter
{
	public static void main(String[] args)
	{
		//Essentially our main method to perform everything else already coded.
		try
		{
			Parser p = new Parser("C:\\Users\\Gibus the Wat\\IdeaProjects\\JuliaToJava\\src\\expr4.txt");
			Program program = p.parse();
			program.execute();
			Memory.displayMemory();
		}
		catch (FileNotFoundException e)
		{
			e.printStackTrace();
		}
	}
}